plugin.js

```diff
@@ -90,14 +93,11 @@ function processWorkletFunction(t, fun) {
   const closure = new Map();
   const outputs = new Set();
 
-  const funScope = fun.scope;
+  // We use copy because some of the plugins don't update bindings and
+  // some even brake them!
+  const astWorkletCopy = parse('\n(' + fun.toString() + '\n)');
 
-  fun.traverse({
-    DirectiveLiteral(path) {
-      if (path.node.value === 'worklet' && path.getFunctionParent() === fun) {
-        path.parentPath.remove();
-      }
-    },
+  traverse(astWorkletCopy, {
```

Comment:
Wouldn't it be safer to extract "outputs" on the parsed copy instead of keeping it in `fun.traverse` block? Then we can keep `fun.traverse` only to do `'worklet'` directive cleanup